apps/test-bundle-button/prettier.config.js

```diff
@@ -0,0 +1,3 @@
+const prettierConfig = require('office-ui-fabric-react-tslint/prettier.config.js');
+
+module.exports = Object.assign({}, prettierConfig);
```

Comment:
maybe we need a deep merge?

Reply:
Not sure, but probably not for now since the config is just

```js
module.exports = {
  "printWidth": 140,
  "tabWidth": 2,
  "singleQuote": true
}; 
```